src/wallet/wallet.cpp

```diff
@@ -1823,6 +1893,14 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend,
                     strFailReason = _("Transaction too large");
                     return false;
                 }
+                
+                //remove signature if we used the signing only for the fee calculation
```

Comment:
We still need to remove the signature, but now, during fundrawtransaction, the signature is dummy/void.